### PR TITLE
Add "Understanding CMS Data Access" proposal

### PR DESCRIPTION
**Pull request by sfiligoi**
Add "Understanding CMS Data Access" proposal.
Also add UCSD as an organization, and Igor Sfiligoi as a mentor.

### CODE REVIEW

**Comment by sfiligoi** [comment]
The CI check seems to have failed on something other than my changes.

**Comment by JavierCVilla** [comment]
Hi @sfiligoi, what would be the percentages of coding and analysis that you expect for this project?

Also, it is recommended to have a co-mentor for each project. Do you think you could add someone with background on this project as a backup?

**Comment by sfiligoi** [comment]
@JavierCVilla I have updated the proposal text to clarify the fact that there is significant coding involved. I have also added a co-mentor.

File: _gsocproposals/2019/proposal_CMSDataAccess.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+ - Analyze the curated data.
+  Put in place tools and procedures that allow for monitoring of deviations from discovered trends in the longer term.  
+ - Put in place a modeling framework for predicting the access patterns within an alternate data access infrastructure. 
+ - Create a model of potential novel infrastructures, and determine the predicted access pattern in it.
+
+## Expected results
+ - A set of tools for cleansing and extracting access pattern data.
+ - A set of tools for extracting access pattern trends from cleansed data.
+ - A modeling framework for predicting access patterns.
+ - A report than validates the usefulness of the tools and frameworks through the analysis of the currently available data
```
Comment (JavierCVilla) [inline _gsocproposals/2019/proposal_CMSDataAccess.md]: Typo? 

A report *than* --> **that** ? validates

Comment (sfiligoi) [reply _gsocproposals/2019/proposal_CMSDataAccess.md]: Yes, that was a typo.
Fixed.

Thanks for catching it.

**Pull request merged**